src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -1223,18 +1223,21 @@ protected NodeModel(IEnumerable<PortModel> inPorts, IEnumerable<PortModel> outPo
             RaisesModificationEvents = true;
         }
 
+
+        internal string cachedAsmName = null;
```

Comment:
I would make this protected

Reply:
ANother name for the member could be 
`NameOfAssemblyReferencedByNode` to match the existing method